src/OrchardCore.Modules/OrchardCore.Users/Extensions/UserTimeZoneServiceExtensions.cs

```diff
@@ -0,0 +1,50 @@
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.DependencyInjection;
+using OrchardCore.Modules;
+using OrchardCore.Users.Models;
+using OrchardCore.Users.TimeZone.Services;
+
+namespace OrchardCore.Users;
+
+/// <summary>
+/// Provides an extension for <see cref="IUserTimeZoneService"/>.
+/// </summary>
+public static class UserTimeZoneServiceExtensions
+{
+    private static HttpContext HttpContext => new HttpContextAccessor().HttpContext;
+
+    /// <summary>
+    /// Gets the time zone for currently logged-in user.
+    /// </summary>
+    /// <param name="userTimeZoneService">The <see cref="IUserTimeZoneService"/>.</param>
+    public static async ValueTask<ITimeZone> GetAsync(this IUserTimeZoneService userTimeZoneService)
+    {
+        ArgumentNullException.ThrowIfNull(nameof(userTimeZoneService));
```

Comment:
```suggestion
        ArgumentNullException.ThrowIfNull(userTimeZoneService);
```

Reply:
`nameof` is never null and the name of the parameter is taken automatically.